srcs/parser/expand/expand.c

```diff
@@ -88,6 +93,10 @@ int			expand(t_lexer *lexer, t_proc *p, int *i)
 			return (ret);
 		if ((ret = expand_glob_brace(lexer->sh, &argv_list)) != ST_OK)
 			return (ret);
+
+		// BUT IT SHOULD BE PLACED HERE :-)
+		if ((ret = globbing(&argv_list)) != ST_OK)
+			return (ret);
```

Comment:
mais plutot ici
par contre, ici, ça génère un "pointer being freed was not allocated"